common/astyanax/src/main/java/com/bazaarvoice/emodb/common/cassandra/CqlCluster.java

```diff
@@ -63,6 +63,26 @@ private void registerMetrics() {
                 metrics.getOpenConnections());
 
         _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "trashed-connections"),
+                metrics.getTrashedConnections());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "executor-queue-depth"),
+                metrics.getExecutorQueueDepth());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "blocking-executor-queue-depth"),
+                metrics.getBlockingExecutorQueueDepth());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "reconnection-scheduler-task-count"),
+                metrics.getReconnectionSchedulerQueueSize());
+
+        _metricRegistry.register(
+                MetricRegistry.name("bv.emodb.cql", _metricName, "ConnectionPool", "task-scheduler-task-count"),
+                metrics.getTaskSchedulerQueueSize());
+
```

Comment:
I agree it's a little odd that some metrics are called "queue-depth" while others are "task-count".  I was following the naming conventions used by Datastax's implementation [here](https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/Metrics.java#L81).